frappe/automation/doctype/auto_repeat/auto_repeat.py

```diff
@@ -372,14 +374,14 @@ def send_notification(self, new_doc):
 		elif "{" in self.message:
 			message = frappe.render_template(self.message, {"doc": new_doc})
 
-		frappe.sendmail(
-			reference_doctype=new_doc.doctype,
-			reference_name=new_doc.name,
+		make(
+			doctype=new_doc.doctype,
+			name=new_doc.name,
 			recipients=self.recipients,
 			subject=subject,
 			content=message,
 			attachments=attachments,
-			expose_recipients="header",
```

Comment:
appropriate changes are needed for this - reverting the change won't be enough

cc: @shariquerik

Reply:
@phot0n elaborate what change is required?

Reply:
already communicated to @shariquerik

#

since this pr has already been merged things which fixed in the previous versions will probably be back again.

Reply:
Mention the "fix" in PR description then, else hard to guess what was fixed.